### DISCUSSION TITLE
GSOC Deadline is April 2

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#66] [Announcements]
You may have seen the outreachy-focused version of this announcement/question (https://github.com/orgs/camicroscope/discussions/840) but I understand that the GSOC deadline is April 2 at 18:00 UTC. We don't have any power over google here, in case that wasn't clear, so be sure to have any proposals you want in on time.

We've switched our model for selection a little bit, and we'd like to know if anything is unclear. Most of it is still the same, the questions reviewers will ask are still : "What is this person proposing", "Does their proposal reflect understanding", and "Does it seem like they have demonstrated the ability to deliver their proposal". The difference, we're hoping, is that contribution tasks will help people learn how to work with caMicroscope's code base specifically a way that the code challenges did not. Also it's hopefully wasting less of your (contributors') time.

(Also we ended up not being able to pass a selected contributor last year since we didn't onboard them to caMicroscope's code base as well as we thought, and that was a touch situation and decision, and we'd like to avoid any possibility of making a hard decision like that again)

You're still of course welcome to point to something you did if you think it helps your application answer the questions above.

If you have any questions/concerns, please follow up on this/another thread in discussions.

Thank you all,
- Birm

**Comment by lbhnsh** [comment]
Hey @birm , once a proposal is made,
Will you be providing insights or reviewing it before submitting it on the GSOC website?
To get insights on improving the proposals.
If yes, can I mail the proposal to the specific mentor(s) for a review?

Thanks

**Comment by bruno-noir** [comment]
Hi, I'm in the process of finalizing my GSOC proposal. Could you let me know if there's an opportunity for review  before submission?
